tests/foreman/ui/test_computeresource_openstack.py

```diff
@@ -0,0 +1,361 @@
+"""Test for compute resource UI
+
+@Requirement: Computeresource
+
+@CaseLevel: Acceptance
+
+@CaseComponent: UI
+
+@TestType: Functional
+
+@CaseImportance: High
+
+@Upstream: No
+"""
+
+from robottelo.decorators import run_only_on, stubbed, tier1, tier2, tier3
+from robottelo.test import UITestCase
+
+
+class OpenstackComputeResourceTestCase(UITestCase):
+    """Implements compute resource tests in UI"""
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier1
+    def test_positive_create_openstack_with_name(self):
+        """Create a new openstack compute resource using different data values
+        as name
+
+        @id: 17f8e903-c2db-4580-8162-0706da6c6368
+
+        @setup: openstack hostname and credentials.
+
+        @steps:
+        1. Create a compute resource of type openstack.
```

Comment:
Thank you , didn't know this , will check my other PR's for the same and update them